docs/home_assistant.md

```diff
@@ -54,7 +54,7 @@ Starting via service is also possible, but you have to change the configuration
 Configuration:
 --------------
 
-The configuration works as described within [Home Assistant documentation](https://home-assistant.io/components/#search/knx) with the difference that the component is called `xknx` instead of `knx`.
+The configuration for the manually checked out version works the same as described within [Home Assistant KNX documentation](https://home-assistant.io/components/#search/knx) with the difference that the component is called `xknx` instead of `knx` (which is be the HASS default KNX integration name).
```

Comment:
- "with the difference that the component is called `xknx`" should we use "platform" instead of component? (Component is now light or switch - I think the wording changed some time ago)
- "which is be" doesn't sound right. I think this is a typo.

Reply:
* The wording in general in HA is quite confusing to me, which is why I triggered a new discussion [here](https://github.com/home-assistant/developers.home-assistant/issues/570) to get more clarity.
* Yes, midnight typo.